front/components/ErrorComponent/ErrorComponent.tsx

```diff
@@ -66,13 +87,15 @@ export const ROUTE = (code?: ErrorComponentProps['code']) =>
 
 export const ErrorComponent = (props: ErrorComponentProps) => {
   return (
-    <div className="error-component">
-      <h2>
-        <FormattedMessage {...messages[props.code].title} />
-      </h2>
-      <p>
-        <FormattedMessage {...messages[props.code].text} />
-      </p>
-    </div>
+    <ErrorComponentStyled>
+      <ErrorContent>
+        <H2>
```

Comment:
the capital lettre in H2 looks weird. Is it the way things are done with styled components?